packages/kernel/py/stlite-server/stlite_server/upload_file_request_handler.py

```diff
@@ -61,8 +62,11 @@ def post(self, request: Request, **kwargs) -> Response:
         )
 
         try:
-            session_id = self._require_arg(args, "sessionId")
-            widget_id = self._require_arg(args, "widgetId")
+            path_args = re.match(UPLOAD_FILE_ROUTE, request.path)
+            session_id = path_args.group('session_id')
+            file_id = path_args.group('file_id')
```

Comment:
TODO: Check if `session_id` and `file_id` can be obtained from `kwargs` of this handler?
(Now I'm outside and can't confirm it, but think maybe it's possible)